packages/consoledot-api/src/transformers/topicTransformer.ts

```diff
@@ -203,3 +204,20 @@ function configValueToBoolean(value: string, defaultIfError: boolean): boolean {
   }
   return defaultIfError;
 }
+
+export const convertToKeyValuePairs = (inputObj: TopicConfig) => {
+  const keyValuePairs: ConfigEntry[] = [];
+  for (const [key, value] of Object.entries(inputObj)) {
+    if (key === "retention.ms") {
+      keyValuePairs.push({
+        key,
+        value: (value as Milliseconds).value.toString(),
+      });
+    } else if (key === "retention.bytes") {
+      keyValuePairs.push({ key, value: (value as Bytes).value.toString() });
+    } else if (key === "cleanup.policy") {
+      keyValuePairs.push({ key, value: value.toString() });
+    }
+  }
+  return keyValuePairs;
+};
```

Comment:
This is smart, but risky. Since the `ConfigEntry` type is very lax, this doesn't give us much in terms of safety of the generated list.

I'd manually create the list explicitly listing all the properties that we are supposed to set, as we have a type `UserEditable` that lists precisely what it is that the UI is supposed to be able to edit.